Sources/ClientRuntime/Networking/Http/SdkHttpRequest+CRT.swift

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import class SmithyHTTPAPI.SdkHttpRequest
+import AwsCommonRuntimeKit
+
+extension SdkHttpRequest {
```

Comment:
`SdkHttpRequest` was moved out to `SmithyHTTPAPI`, but this extension remains for the CRT dependency.